auth/v1.js

```diff
@@ -57,10 +57,12 @@ var api = new API({
     table:      /^[A-Za-z][A-Za-z0-9]{2,62}$/,
 
     // Patterns for AWS
-    bucket:     /^[a-z0-9][a-z0-9-]{1,62}[a-z0-9]$/
+    bucket:     /^[a-z0-9][a-z0-9-]{1,62}[a-z0-9]$/,
                 // we could allow "." too, but S3 buckets with dot in the name
                 // doesn't work well with HTTPS and virtual-style hosting.
                 // Hence, we shouldn't encourage people to use them
+    // Project for sentry (and other per project resources)
+    project:    /^[a-zA-Z0-9_-]{1,22}$/,
```

Comment:
This is just the name of the pattern when it shows up in a url pattern. Nothing more...
It carries no semantics... And outside of the API end-points where it is used it's not visible.

I would like to keep it as project, because I'm adding another end-point that will be using the same pattern, and the same name "project", for a url-pattern parameter.